src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -15,36 +18,186 @@ namespace Microsoft.Sbom.Utils;
 /// </summary>
 public static class SPDXToSbomFormatConverterExtensions
 {
-    /// <summary>
-    /// Converts a <see cref="SPDXFile"/> object to a <see cref="SbomFile"/> object.
-    /// </summary>
-    /// <param name="spdxFile"></param>
-    /// <returns></returns>
-    public static SbomFile ToSbomFile(this File spdxFile)
+    public static SbomFile ToSbomFile(this File spdxFile, List<Element> spdx30Elements = null, List<Relationship> relationships = null)
     {
-        var checksums = spdxFile.VerifiedUsing?.Select(c => c.ToSbomChecksum());
-
-        // Not setting LicenseConcluded and LicenseInfoInFiles since the whole SBOM is required to set these values.
         return new SbomFile
         {
-            Checksum = checksums,
-            FileCopyrightText = spdxFile.CopyrightText,
+            Checksum = spdxFile.VerifiedUsing.ToSbomChecksum(),
```

Comment:
If I'm reading this right, it looks like we used to allow spdxFile.VerifiedUsing to be null (see the ? in the old code) but now it's required to be non-null. Is that an intentional change? Also why is the .Select no longer necessary?

Reply:
I removed the ? in favor of adding a null check in the helper method instead. Select is no longer necessary because I am returning a List<SbomChecksum> rather than a singular SbomChecksum.

Reply:
Correction on the first part: It looks like in SPDX 2.2 extensions we use ? rather than adding a null check in the helper method, so I will add back the ? to maintain consistency. Also in the helper method, I returned null if VerifiedUsing.Count == 0, but this is not consistent with SPDX 2.2 extension. Rather than returning null it should return an empty checksum list.

I will also add a unit test to validate this behavior for checksums.